pyproject.toml

```diff
@@ -1,3 +1,157 @@
+[build-system]
+requires = ["setuptools >= 61.1.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "optuna"
+description = "A hyperparameter optimization framework"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+  {name = "Takuya Akiba"},
+  {email = "akiba@preferred.jp"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.7"
+dependencies = [
+  "alembic>=1.5.0",
+  "cmaes>=0.9.1",
+  "colorlog",
+  "numpy",
+  "packaging>=20.0",
+  "sqlalchemy>=1.3.0",
+  "tqdm",
+  "PyYAML",  # Only used in `optuna/cli.py`.
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+benchmark = [
+  "asv>=0.5.0",
+  "botorch",
+  "cma",
+  "scikit-optimize",
+  "virtualenv"
+]
+checking = [
+    "black",
+    "blackdoc",
+    "hacking",
+    "isort",
+    "mypy",
+    "types-PyYAML",
+    "types-redis",
+    "types-setuptools",
+    "typing_extensions>=3.10.0.0",
+]
+document = [
+    "cma",
+    "distributed",
+    "fvcore",
+    "lightgbm",
+    "matplotlib!=3.6.0",
+    "mlflow",
+    "pandas",
+    "pillow",
+    "plotly>=4.9.0",  # optuna/visualization.
+    "scikit-learn",
+    "scikit-optimize",
+    "sphinx<6",
```

Comment:
Missed the todo comment above this line. 

`# TODO(not522): Remove the constraint after sphinx_rtd_theme supports Sphinx 6`

Reply:
:bow: fixed in f81457c.